cmd/pack3d/main.go

```diff
@@ -28,14 +30,20 @@ func timed(name string) func() {
 }
 
 func main() {
+	var stlsize []fauxgl.Vector
```

Comment:
maybe  a better name would be 'naive_stl_size' or 'simple_stl_size' ? We should indicate it's a 'dumb' size of just adding max dimensions. 'dumb_stl_size' ?